Formula/l/ldb.rb

```diff
@@ -0,0 +1,67 @@
+class Ldb < Formula
+  desc "LDAP-like embedded database"
+  homepage "https://ldb.samba.org"
+  url "https://download.samba.org/pub/ldb/ldb-2.9.1.tar.gz"
+  sha256 "c95e4dc32dea8864b79899ee340c9fdf28b486f464bbc38ba99151a08b493f9b"
+  license all_of: ["GPL-3.0-or-later", "LGPL-3.0-or-later"]
+
+  livecheck do
+    url "https://download.samba.org/pub/ldb/"
+    regex(/href=.*?ldb[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "cmocka" => :build
+  depends_on "pkg-config" => :build
+  depends_on "lmdb"
+  depends_on "popt"
+  depends_on "talloc"
+  depends_on "tdb"
+  depends_on "tevent"
+
+  uses_from_macos "python" => :build
+
+  def install
+    args = %W[
+      --bundled-libraries=NONE
+      --disable-python
+      --disable-rpath
+      --disable-rpath-private-install
```

Comment:
May need one of these for Linux.

Reply:
Yeah, the install RPATHs are needed. Luckily the RPATHs seem to work on Linux. Most of disabling is odd RPATHs that get created on macOS, like the non-existent `libtdb.inst.dylib`